bear/main.py.in

```diff
@@ -435,6 +435,8 @@ def split_command(command):
                 next(args)
         elif re.match(r'^-(l|L|Wl,).+', arg):
             pass
+        elif re.match(r'^-Wp,-M.+', arg):
```

Comment:
Interesting! :) Can you hint me which build system does that? (Or which project use it?) Also can you show me a usage?
And might put into the `IGNORED_FLAGS` dictionary rather than create a regex for it. What do you think?